tornado/test/log_test.py

```diff
@@ -160,6 +161,37 @@ def test_log_file(self):
                 os.unlink(filename)
             os.rmdir(tmpdir)
 
+    def test_log_file_with_timed_rotating(self):
+        tmpdir = tempfile.mkdtemp()
+        try:
+            self.options.log_file_prefix = tmpdir + '/test_log'
+            self.options.rotating_mode = 'timed'
+            self.options.log_file_rotating_when = 'S'
+            self.options.log_file_rotating_interval = 1
+            enable_pretty_logging(options=self.options, logger=self.logger)
+            self.logger.error('hello1 from TimedRotatingFileHandler')
+            time.sleep(1.01)
```

Comment:
All our other tests complete in far less than 1 second; I don't think it's worthwhile to do this here. Since the real rotation code is in the `logging` module and we're just doing a little configuration, I think it's fine not to test the actual rollover. Just do the same sort of test as `test_log_file` to make sure that the handler is configured and logs can be written to disk.